plugin/web/extensions/sidebar/popups/mobileDropdown/Catalog/index.tsx

```diff
@@ -41,9 +44,11 @@ const Catalog: React.FC<Props> = ({
   setSelectedDataset,
   setExpandedFolders,
   onSearch,
+  setSearchTerm,
   onDatasetAdd,
 }) => {
   const [filter, setFilter] = useState<GroupBy>("city");
+  const [currentTreeTab, changeTreeTab] = useState<TreeTab>("city");
```

Comment:
I don't think we need this. It is identical to filter, right? They seem to always be synced up, so just use filter where you are using treeTab.

Reply:
Done!